README.md

```diff
@@ -64,6 +75,17 @@ module: {
                 //         naming: 'origin'
                 //     }
                 // },
+                // OR for a few entires:
+                // levels: [
+                //     [
```

Comment:
How can you match this with named entry points?

Reply:
I did not find out a way to match them. So i made the file structure like this 
```
│
└─── blocks1
│   │   entry-point1.js
│   │   ...
│   
└─── blocks2
│   │   entry-point2.js
│   │   ...
│   
└─── overrides
│   │
│   └─── overrides-blocks1
│   │   │   ...
│   │
│   └─── overrides-blocks2
│   │   │   ...
```

The entry points
```
entry: {
    entry1: 'blocks1/entry-point1.js',
    entry2: 'blocks2/entry-point2.js'
}
```

The loader config
```
loader: 'webpack-bem-loader',
options: {
    ...
    levels: [
        [
            'blocks1',
            'overrides/overrides-blocks1'
        ],
        [
            'blocks2',
            'overrides/overrides-blocks2'
        ]
    ],
    ...
}
```

Reply:
@awinogradov which changes you believe are to be implemented?